site/content/user-guide/amqp-adapter.md

```diff
@@ -0,0 +1,175 @@
+
+In the case of a server-side error, the adapter responds with a *RELEASED* disposition, which indicates to the client that the message was OK but it cannot be delivered due to a failure beyond the control of the client. In this case, the client should attempt to redeliver the message again.
+
+<!--more-->
+
+## Device Authentication
+
+By default, all Hono protocol adapters require clients (devices or gateway components) to authenticate during connection establishment. This is the preferred way for devices to publish data via protocol adapters. The AMQP adapter can be configured to *disallow* authentication for devices by setting the value of `HONO_AMQP_AUTHENTICATION_REQUIRED` to `false`. In this guide, we will give examples for publishing telemetry and events for *authenticated* and *unauthenticated* clients. We will see in the examples later that the main difference between publishing messages for authenticated and unauthenticated devices is in the format of the AMQP 1.0 **address**. For unauthenticated devices, the address should include the tenant and device identifier.
+
+### SASL PLAIN Authentication
+
+The AMQP adapter currently supports authenticating clients using a *username* and *password*. This means that clients needs to provide a *username* and a *password* when connecting to the AMQP adapter. The *username* must match the pattern [*auth-id@tenant*], e.g. `sensor1@DEFAULT_TENANT`. The adapter verifies the credentials provided by the client against the credentials that the [Credentials Service] ({{< relref "#credentials-service-connection-configuration" >}}) has on record for the device. If the credentials match, then authentication is successful and the client device can proceed to publish messages to Hono.
+
+When running the Hono example installation, as described in the [Getting Started guide]({{< relref "getting-started.md" >}}), the demo Credentials service comes pre-configured with a `hashed-password` secret for devices `4711` and `gw-1` of tenant `DEFAULT_TENANT` as shown below. These credentials are used in the following examples to illustrate the usage of the adapter. We use the **qpid-send** command-line tool (from the Apache Qpid project) to simulate a device publishing messages to the AMQP adapter.
+
+{{% note %}}
+There is a subtle difference between the *device identifier* (*device-id*) and the *auth-id* a device uses for authentication. See [Device Identity]({{< relref "concepts/device-identity.md" >}}) for a discussion of the concepts.
+{{% /note %}}
+
+### SASL EXTERNAL Authentication
+TODO
+
+## Publish Telemetry Data (authenticated Device)
+
+The AMQP adapter supports publishing of telemetry to Hono's Telemetry API. Telemetry messages are published using both `AT_LEAST_ONCE` and `AT_MOST_ONCE` delivery semantics.
+
+* Address: `telemetry`
+* Delivery: `AT_MOST_ONCE` or `AT_LEAST_ONCE`
```

Comment:
These terms are not defined by AMQP 1.0. Instead, you need to define these delivery modes by means of *presettled* vs *unsettled* ...